velox/docs/functions/spark/misc.rst

```diff
@@ -2,6 +2,11 @@
 Miscellaneous Functions
 ====================================
 
+.. spark:function:: at_least_n_non_nulls(n, value1, value2, ..., valueN) -> bool
+
+    Returns true if there are at least ``n`` non-null and non-NaN values, 
+    or false otherwise. If ``n <= 0``, the result is true.
+
```

Comment:
nit: perhaps add an example that covers both NULL and NaN.

Reply:
sure, added.